### DISCUSSION TITLE
Is there a way to download DRM protected-MP4 videos from OnlyFans?

### DISCUSSION THREAD

**Discussion by DuneCyber90** [aclap-dev/video-downloadhelper#256] [Help and Q&A]
### Your question

As is mentioned above Is there a way to download DRM protected-MP4 videos from OnlyFans? As I have tried to download videos that I have purchased from Onlyfans only to find that they are not able to be downloaded. Is there a way round this?

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-GB
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\mjtre",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
Not with VDH.  I don't know about what might happen with other tools.

**Comment by mikegab13** [comment]
The program EaseUS RecExperts allows you to screen record DRM protected videos.  You have to pay for it though....the free version only allows up to 1 minute of recording.

**Comment by LJD1800** [comment]
I've never tried it with a purchased video but I have found that VDH has worked with a paid subscription.

**Reply by GrampaWildWilly** [reply]
Having a VDH license isn't really the issue.  Having a paid subscription to a given web site isn't the issue either, nor is the one-off purchase of a given video.  DRM is the issue.  Digital Rights Management.  Think copyrights & lawyers.  People who own assets from which they wish to make money are usually interested in actually making that money.  If their content is protected by DRM, that is one way to monetize the asset.  If you want the item, you have to pay for it.  The owner probably doesn't want you to be able to make a copy because then you could give it to any number of other people who would not pay for it like you did.  Of course, I thought this was all decided 50 or so years ago with the advent of consumer availability of the VCR, which allowed people to record their TV shows to time-shift their watching to their convenience rather than the convenience of network programming executives.  I find DRM protection annoying.  I recently investigated a number of screen recorders & it seems the lawyers have gotten to almost all of those people.  Most of the screen recorders I looked at respect DRM now.  This did not used to be true, but it is now.  The 2 primary screen recorders that respect DRM are OBS & the screen snipper tool built into Windows 11.  Some day, if I get sufficiently annoyed, I may resume my search for a screen recorder that gets around DRM.  Maybe I'll have to pay for it.  As things stand now, I am not there yet & I feel it's unlikely I'll ever get there.  Go ahead.  DRM your stuff.  I'll ignore it.  I'll live.

**Reply by LJD1800** [reply]
I'm aware of DRM & what it does.  I was just pointing out that I have no problem using DVH on OF when I use it with a paid subscription and not "one-off purchase" as you are doing.  
You might look into Audials.  I've never tried it with OF but it gets me around the BRM & works well at other sites.
